nixos/modules/services/x11/display-managers/startx.nix

```diff
@@ -39,6 +39,10 @@ in
       displayManager.lightdm.enable = lib.mkForce false;
     };
     systemd.services.display-manager.enable = false;
+
+    # Let Xorg use its default log location ($XDG_DATA_HOME/xorg/)
```

Comment:
I'm a bit confused, isn't the default after all previous changes `/var/log/xorg`?

Reply:
Xorg logs to `$XDG_DATA_HOME/xorg/` when run as a regular user and `/var/log/xorg` when run as root.

I enhanced the commit messages to include this info.